include/forms/textBoxes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+function singleLineTextBox($textTitle,$textName,$textPlaceholder, $value, $valueArray){
```

Comment:
Later on you might want to be able to pass in additional style (so you can change the look of specific form elements) or a class so that CSS can be applied to it. Don't worry about adding that until you need it though.